### PR TITLE
First attempts to add rsync example to ci run

### PR DESCRIPTION
**Pull request by JoshuaSBrown**
This pr is designed to incorporate the examples as part of the CI. A GitHub Action has been created that will run the imagemagick example problem.

### CODE REVIEW

File: tests/test_dummy.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import pytest
+
+
+@pytest.mark.integration
+def test_dummy1_true():
+    assert True
```
Comment (JoshuaSBrown) [inline tests/test_dummy.py:6]: What is this part for?

File: tests/test_imagemagick_files.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    # # Step 4 launch example
+    # process = subprocess.Popen(
+    #     command,
+    #     shell=False,
+    #     stdout=subprocess.DEVNULL,
+    #     stderr=subprocess.STDOUT)
+    #
+    # stdout, stderr = process.communicate()
+    # print(stdout)
+    # print(stderr)
```
Comment (JoshuaSBrown) [inline tests/test_imagemagick_files.py:48]: We can delete this.

Comment (JoshuaSBrown) [reply tests/test_imagemagick_files.py:48]: I would add a note why subprocess doesn't work.

**Comment by JoshuaSBrown** [comment]
@renan-souza this pr was merged into devel right, if so can you close this PR.

File: tests/test_imagemagick_files.py (hunk 2 above)
Comment (renan-souza) [reply tests/test_imagemagick_files.py]: These are leftovers from a debugging I was doing. It is outdated. As of the HEAD version of this branch [JoshuaSBrown-ci-integration-main](https://github.com/ORNL/zambeze/tree/JoshuaSBrown-ci-integration-main), I changed that and left all uncommented. Subprocess does work.

**Pull request closed without merging**